src/equations/compound-interest.js

```diff
@@ -1,12 +1,37 @@
-export const compoundMaturity = (
-  principal,
-  interestRate,
-  periods
-) => {
-  // compoundsPerYear
+const pow = Math.pow
+const exp = Math.exp
 
-  return principal * Math.exp(interestRate*periods);
+export default class Config {
+  static title = "Compound Interest"
+  static description = "Standard PeRT Equation."
 
+  static params = {
+    principal: {
+      label: "Principal",
+      default: 1000,
+      type: "currency",
+      min: 0,
+      max: Number.POSITIVE_INFINITY,      
+    },
+    periods: {
+      label: "Periods",
+      default: 10,
+      type: "currency",
```

Comment:
we need to create new types because this isn't really a currency. this would just be a `number` type. and you'll need to set a min/max here, too

Reply:
i would like to make defaults for min max and such. i don't think it makes sense to oblige every value to be filled in. Working on a separate PR for that.  Will fix the type. lmk